Astronomy/Astronomy/Helper Files/FetchPhotoOperation.m

```diff
@@ -0,0 +1,85 @@
+//
+//  FetchPhotoOperation.m
+//  Astronomy
+//
+//  Created by Karen Rodriguez on 5/19/20.
+//  Copyright © 2020 Wyatt Harrell. All rights reserved.
+//
+
+#import "FetchPhotoOperation.h"
+#import "Astronomy-Swift.h"
+
+@interface FetchPhotoOperation()
+
+@property BOOL executing;
+@property BOOL finished;
+//@property BOOL cancelled;
```

Comment:
You don't need to override cancelled, you can use the default behavior. Read docs on it.